test/pkg/testconfig/testconfig.go

```
@@ -890,14 +991,29 @@ func CreatePtpConfigWPCGrandMaster(policyName string, nodeName string, ifList []
+		return fmt.Errorf("failed to get test config: %v", err)
+	}
+
+	// Get E810 plugin settings from environment variables (following existing pattern)
+	// MAX_OFFSET_IN_NS = LocalMaxHoldoverOffset, MIN_OFFSET_IN_NS = -LocalMaxHoldoverOffset
+	// HOLDOVER_TIMEOUT_S = LocalHoldoverTimeout (keep from YAML for now)
+	// MAX_IN_SPEC_OFFSET_NS = MaxInSpecOffset
+	localMaxHoldoverOffset := metrics.MaxOffsetNs
+	localHoldoverTimeout := testParameters.GlobalConfig.HoldOverTimeout // Keep this from YAML for now
+	maxInSpecOffset := metrics.MaxInSpecOffsetNs
+
 	plugins := make(map[string]*apiextensions.JSON)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to plugins

(ineffassign)